projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -37,35 +37,27 @@ def start_job_execution(self) -> None:
         API path: POST /data-jobs/for-team/{team_name}/jobs/{job_name}/deployments/{deployment_id}/executions
 
         :return: None
```

Comment:
I don't see explicit `None` return. Maybe this can be removed from the docstring?

Reply:
noted